auv_sim/auv_sim_description/auv_common_sim_descriptions/urdf/sensors/xsens_mti_g710.urdf.xacro

```diff
@@ -1,12 +1,34 @@
 <?xml version="1.0"?>
 <robot name="xsens_mti_g710" xmlns:xacro="http://www.ros.org/wiki/xacro">
   <xacro:macro name="xsens_mti_g710_sim" params="name namespace static_reference_frame">
+    <link name="${name}_sim_link">
+      <inertial>
+        <mass value="0.001" />
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <inertia ixx="0.0000001" ixy="0" ixz="0" iyy="0.0000001" iyz="0" izz="0.0000001" />
+      </inertial>
+      <collision>
+        <geometry>
+          <cylinder length="${0.000001}" radius="${0.000001}" />
+        </geometry>
+        <origin xyz="0 0 0" rpy="0 ${0.5*pi} 0" />
+      </collision>
+    </link>
+
+    <joint name="${name}_sim_joint" type="revolute">
+      <parent link="${name}_link" />
+      <child link="${name}_sim_link" />
+      <origin xyz="0 0 0" rpy="${math.pi} ${math.pi} 0" />
+      <axis rpy="0 0 0" xyz="0 0 1" />
+      <limit lower="0.0" upper="0.0" effort="0.0" velocity="0.0" />
+    </joint>
+
     <gazebo>
       <plugin filename="libuuv_gazebo_ros_imu_plugin.so"
         name="${name}_imu_plugin">
         <robot_namespace>${namespace}</robot_namespace>
-        <link_name>${name}_link</link_name>
-        <sensor_topic>sensors/imu/data</sensor_topic>
+        <link_name>${name}_sim_link</link_name>
+        <sensor_topic>sim/imu</sensor_topic>
         <gyroscope_noise_density>0.0002514074</gyroscope_noise_density>
```

Comment:
Are these ip to date with the imu's actual parameters or our estimation?

Reply:
if you meant the noise values, they are the same as our legacy codebase. I can find the noise density values in the sensor specifications but other values like random_walk, bias_correlation_time and turn_on_bias_sigma are not specified in the documents. Not sure what are those or how can we estimate them. This is probably worth an investigation.

Reply:
I think this PR's context can be kept around adding interfaces and links for the sensors, we can address those later.